File: Java/Events/AvailableTicketsBlockInfo.java
package LTD_API;

import java.util.List;

public class AvailableTicketsBlockInfo
{
    private String areaName;
    private decimal facePrice;
    private decimal sellingPrice;
    private List<AvailableTicketInfo> availableTicketsInfos;
    
    public String getAreaName()
    {
        return areaName;
    }
    public void setAreaName(String areaName)
    {
        this.areaName = areaName;
    }

    public decimal getFacePrice()
    {
        return facePrice;
    }
    public void setFacePrice(decimal facePrice)
    {
        this.facePrice = facePrice;
    }

    public decimal getSellingPrice()
    {
        return sellingPrice;
    }
    public void setSellingPrice(decimal sellingPrice)
    {
        this.sellingPrice = sellingPrice;
    }

    public List<AvailableTicketInfo> getAvailableTicketsInfos()
    {
        return availableTicketsInfos;
    }
    public void setAvailableTicketsInfos(List<AvailableTicketInfo> availableTicketsInfos)
    {
        this.availableTicketsInfos = availableTicketsInfos;
    }
}

File: PHP/Events/AvailableTicketsBlockInfo.php
<?php
namespace LTD_API;

class AvailableTicketsBlockInfo
{
    private $areaName;
    private $facePrice;
    private $sellingPrice;
    //array of AvailableTicketInfo
    private $availableTicketsInfos;
    
    public function getAreaName()
    {
        return $this->areaName;
    }
    public function setAreaName($areaName)
    {
        $this->areaName = $areaName;
    }

    public function getFacePrice()
    {
        return $this->facePrice;
    }
    public function setFacePrice($facePrice)
    {
        $this->facePrice = $facePrice;
    }

    public function getSellingPrice()
    {
        return $this->sellingPrice;
    }
    public function setSellingPrice($sellingPrice)
    {
        $this->sellingPrice = $sellingPrice;
    }

    public function getAvailableTicketsInfos()
    {
        return $this->availableTicketsInfos;
    }
    public function setAvailableTicketsInfos($availableTicketsInfos)
    {
        $this->availableTicketsInfos = $availableTicketsInfos;
    }
}

